application/src/main/java/org/thingsboard/server/controller/RuleChainController.java

```diff
@@ -288,24 +257,11 @@ public RuleChain saveRuleChain(
     public RuleChain saveRuleChain(
             @ApiParam(value = "A JSON value representing the request.")
             @RequestBody DefaultRuleChainCreateRequest request) throws ThingsboardException {
-        try {
-            checkNotNull(request);
-            checkParameter(request.getName(), "name");
-
-            RuleChain savedRuleChain = installScripts.createDefaultRuleChain(getCurrentUser().getTenantId(), request.getName());
-
-            tbClusterService.broadcastEntityStateChangeEvent(savedRuleChain.getTenantId(), savedRuleChain.getId(), ComponentLifecycleEvent.CREATED);
 
-            logEntityAction(savedRuleChain.getId(), savedRuleChain, null, ActionType.ADDED, null);
-
-            return savedRuleChain;
-        } catch (Exception e) {
-            RuleChain ruleChain = new RuleChain();
-            ruleChain.setName(request.getName());
-            logEntityAction(emptyId(EntityType.RULE_CHAIN), ruleChain, null, ActionType.ADDED, e);
-            throw handleException(e);
-        }
-    }
+        checkNotNull(request);
+        checkParameter(request.getName(), "name");
+        return tbRuleChainNotifyService.saveRuleChain(getTenantId(), request, getCurrentUser());
```

Comment:
in this case **saveRuleChain** method is a little confused. **RuleChain** word is redundant in the current approach.
According to the top interface (SimpleTbEntityService), now it should be **save**. 
I would rename it to **saveByDefaultRequest** or **saveDefaultByName**.

Reply:
Ok